lib/msal-browser/FAQ.md

```diff
@@ -156,3 +158,9 @@ Please see the documentation on [Tenancy in Azure Active Directory](https://docs
 ## My application has multiple resources it needs to access to. How should I handle scopes for access tokens?
 
 Please see the doc about resources and scopes [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/resources-and-scopes.md)
+
+# B2C
+
+## Why is `getAccountByUsername()` returning null, even though I'm signed in?
+
+In order to use `getAccountByUsername()` in B2C scenarios you must enable your `idTokens` to return the `emails` claim in your B2C tenant. MSAL will fill the `username` field on the `AccountInfo` object with the first element of the array returned on the `emails` claim. In most cases this array will only have one element, however, if you notice that your idTokens are returning more than one email on this claim, ensure you are calling `getAccountByUsername` with the first email.
```

Comment:
Is there documentation we can link to for enabling this claim for B2C?

Reply:
I don't think so, but I think it should be added. cc @nickgmicrosoft

Reply:
Added a short description underneath but would be great to have a more generic doc on docs.microsoft.com that applies to more than just this specific use case.